pom.xml

```diff
@@ -233,6 +233,10 @@
             <version>2.6</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
@yegor256 but Takes has this dependency anyway, right now as transitive from `com.jcabi:jcabi-http`